plugins/formal-verification/testData/diagnostics/no_contracts/custom_collection.kt

```diff
@@ -0,0 +1,12 @@
+// WITH_STDLIB
+
+class IntEntry(override val key: Int, override val value: Int): Map.Entry<Int, Int>
+
+// demonstrates that two different 'size' fields are indeed created
```

Comment:
I think this is actually a bug on our side—we create a phantom `size` for reasoning purposes, but then also create a field here because it's explicitly defined.  But in fact these are the same field, and we *shouldn't* be creating both of them.